gui/pages/kernel_options.go

```diff
@@ -0,0 +1,292 @@
+func NewConfigKernelPage(controller Controller, model *model.SystemInstall) (Page, error) {
+	kernelArgsHelp :=
+		utils.Locale.Get("Note: The boot manager tool will first include the \"Add Extra Arguments\" items.")
+	kernelArgsHelp = kernelArgsHelp + "\n" +
+		utils.Locale.Get("Then, the \"Remove Arguments\" items are removed.")
+	kernelArgsHelp = kernelArgsHelp + "\n" +
+		utils.Locale.Get("The final argument list contains the kernel bundle's configured arguments and the ones configured by the user.")
+
+	data, err := kernel.LoadKernelList()
+	if err != nil {
+		return nil, err
+	}
+	page := &ConfigKernelPage{
+		controller: controller,
+		model:      model,
+		data:       data,
+	}
+
+	// Box
+	page.box, err = setBox(gtk.ORIENTATION_VERTICAL, 0, "box-page")
+	if err != nil {
+		return nil, err
+	}
+
+	// ScrolledWindow
+	page.scroll, err = setScrolledWindow(gtk.POLICY_NEVER, gtk.POLICY_AUTOMATIC, "scroller")
+	if err != nil {
+		return nil, err
+	}
+	page.scroll.SetMarginStart(common.StartEndMargin)
+	page.scroll.SetMarginEnd(common.StartEndMargin)
+	page.box.PackStart(page.scroll, true, true, 0)
+
+	// ListBox
+	page.list, err = setListBox(gtk.SELECTION_SINGLE, true, "list-scroller")
+	if err != nil {
+		return nil, err
+	}
+	if _, err := page.list.Connect("row-activated", page.onRowActivated); err != nil {
+		return nil, err
+	}
+	page.scroll.Add(page.list)
+
+	// Create list data
+	for _, v := range page.data {
+		name := v.Name
+		desc := v.Desc
+
+		//Add default label to kernel already in model
+		if page.model.Kernel != nil {
+			if v.Bundle == page.model.Kernel.Bundle {
+				name = v.Name + utils.Locale.Get(" (default)")
+			}
+		}
+
+		box, err := setBox(gtk.ORIENTATION_VERTICAL, 0, "box-list-label")
+		if err != nil {
+			return nil, err
+		}
+
+		labelName, err := setLabel(name, "list-label-name", 0.0)
+		if err != nil {
+			return nil, err
+		}
+		box.PackStart(labelName, false, false, 0)
+
+		labelDesc, err := setLabel(desc, "list-label-desc", 0.0)
+		if err != nil {
+			return nil, err
+		}
+		box.PackStart(labelDesc, true, true, 0)
+
+		page.list.Add(box)
+	}
+
+	// addLabel: Tell users they can add kernel command lines below
+	addText := utils.Locale.Get("Add Extra Arguments:")
+
+	page.addLabel, err = setLabel(addText, "label-addLabel", 0.0)
+	if err != nil {
+		return nil, err
+	}
+	page.addLabel.SetMarginStart(common.StartEndMargin)
+	page.addLabel.SetHAlign(gtk.ALIGN_START)
+	page.box.PackStart(page.addLabel, false, false, 10)
+
+	// addEntry: Args to add to the kernel command line
+	page.addEntry, err = setEntry("addEntry")
+	if err != nil {
+		return nil, err
+	}
+	page.addEntry.SetMarginStart(common.StartEndMargin)
+	page.addEntry.SetMarginEnd(common.StartEndMargin)
+	page.addEntry.SetTooltipText(utils.Locale.Get(kernelArgsHelp))
+	page.box.PackStart(page.addEntry, false, false, 0)
+
+	// remLabel label
+	remText := utils.Locale.Get("Remove Arguments:")
+	page.remLabel, err = setLabel(remText, "label-remLabel", 0.0)
```

Comment:
Please change the style to the "label-entry" which we are using for the add user screen for consistency.

Reply:
Done.